packages/open-payments/src/client/token.ts

```diff
@@ -10,6 +10,7 @@ interface RotateRequestArgs {
 
 export interface TokenRoutes {
   rotate(args: RotateRequestArgs): Promise<AccessToken>
+  revoke(args: RotateRequestArgs): Promise<void>
```

Comment:
Should `RotateRequestArgs` be renamed?

Reply:
Well, seems like they are using same args, and we'd better rename this rather than create same struct.
Maybe rename to TokenRequestArgs? 
How do you think?

Reply:
`TokenRequestArgs` is good